src/components/ContentWrapper.jsx

```diff
@@ -73,8 +91,12 @@ const ContentWrapper = (props) => {
   const [backendStatusRequested, setBackendStatusRequested] = useState(false);
 
   useEffect(() => {
-    if (!experimentId) return;
-    if (!backendLoading) dispatch(loadBackendStatus(experimentId));
+    if (!currentExperiment) return;
+    // clear the store only if we navigate to a new experiment from data-management
```

Comment:
What happens if we navigate to another experiment directly from the URL?

Reply:
If we are not in data-management it won't get reset, so we reset it only if the user is in data-management that way we know they are launching a project.